cirno/Geometry/Circle.cs

```diff
@@ -59,6 +59,52 @@ public Vector GetClosestPoint(Vector point)
             return d1 < d2 ? v1 : v2;
         }
 
+        public float Distance(Vector point)
```

Comment:
Your `Distance` method is exactly same as `GetClosestPoint` method.
Can you remove duplicates?

```csharp
public float Distance(Vector point) {
    return Vector.Distance(point, GetClosestPoint(point));
}
```

There is no `Vector.Distance`, so you should implement it.

Reply:
OK. I will remove later. Having dinner. =D